docs/api.md

```diff
@@ -159,6 +161,110 @@ Content:
 }
 ```
 
+
+### <a name="get-rating"></a>Get Rating
+
+Get a comments rating.
+
+#### URL
+
+`/lectures/:lecture-id/comments/:comment-id/rating`
+
+#### Method
+
+GET
+
+#### URL Parameters
+
+##### Required
+
+* `lecture-id=[integer]` ID of lecture.
+* `comment-id=[integer]` ID of comment to retrieve its rating.
+
+#### Success Respones
+
+##### Success
+
+Code: 200
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "rating": {
+            "description": "The rating of the comment",
+            "type": "number"
+        }
+    }
+}
+```
+
+
+### <a name="rate-comment"></a>Rate a comment
+
+Upvote or downvote a comment
+
+#### URL
+
+`/lectures/:lecture-id/comments/:comment-id/rating/:rating-id`
+
+#### Method
+
+POST
+
+#### URL Parameters
+
+##### Required
+
+* `lecture-id=[integer]` ID of lecture to vote on a comment.
+* `comment-id=[integer]` ID of comment to vote on.
+* `rating-id=[integer]` ID of rating (simulating user).
+
+#### Data Parameters
+
+* `data=[integer]` The number 1 or -1 for upvoting or downvoting.
+
+#### Success Respones
+
+##### Success
+
+Code: 200
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "rating": {
+            "description": "The new rating of the comment",
+            "type": "number"
+        }
+    }
+}
+```
+
+##### Invalid Data
```

Comment:
This should be part of __Error Responses__, not __Success Responses__. Also, add a 404 response when not finding lecture, comment or rating.